lib/instrumentation/modules/http2.js

```diff
@@ -80,6 +80,11 @@ module.exports = function (http2, agent, { enabled }) {
       var status = headers[':status'] || 200
       trans.result = 'HTTP ' + status.toString()[0] + 'xx'
       trans.res.statusCode = status
+      if (status >= 500) {
+        trans.outcome = 'failure'
+      } else {
+        trans.outcome = 'success'
+      }
       trans.res.headers = mergeHeaders(trans.res.headers, headers)
```

Comment:
Here I see that during our handling of an http2 response we're setting the transaction's outcome to `failure` if the http status code is greater than or equal to 500.  This behavior matches the spec.